pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -4364,6 +4365,7 @@ CompletableFuture<Message<byte[]>> examineMessageAsync(String topic, String init
      *
      * @param topic
      * @param applied
+     *            Whether to return the configuration value of the previous level if the current level's don't set
```

Comment:
Is this update related to this pull request? If not, please remove from this PR and improve the javadoc in a separate PR.

Reply:
I corrected the problem and submitted a new Pull Request. <https://github.com/apache/pulsar/pull/23695>